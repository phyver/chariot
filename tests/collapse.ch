:testcollapse (f (C x (C y l))).T.T

:testcollapse (f (C x (C y (C z l)))).T.T.T

:testcollapse (f (C x (C y (C z (C u l))))).T.T.T.T


