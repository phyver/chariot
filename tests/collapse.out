--   collapse of   ((f (C⁼ x (C⁼ y l))).T⁼).T⁼   is   ((f (C⁼ x (C⁼ y l))).T⁼).T⁼

--   collapse of   (((f (C⁼ x (C⁼ y (C⁼ z l)))).T⁼).T⁼).T⁼   is   <1>⁼ . (((f (C⁼ x (C⁼ y <1>⁼ l + <1>⁼ z))).T⁼).T⁼)

--   collapse of   ((((f (C⁼ x (C⁼ y (C⁼ z (C⁼ u l))))).T⁼).T⁼).T⁼).T⁼   is   <∞>⁼ . (((f (C⁼ x (C⁼ y <∞>⁼ l + <∞>⁼ u + <1>⁼ z))).T⁼).T⁼)

